File: Completed/js/absoluteValuesSumMinization/absoluteValuesSumMinization.js
function absoluteValuesSumMinimization(a){
    return (a.length % 2) ? a[Math.floor(a.length / 2)] : a[a.length / 2 - 1]
}

console.log(absoluteValuesSumMinimization([2, 4, 7]));
console.log(absoluteValuesSumMinimization([2, 4, 6, 7]));
console.log(absoluteValuesSumMinimization([2, 4, 6, 6, 7]));
console.log(absoluteValuesSumMinimization([2, 4, 6, 6, 7, 8]));

File: Completed/js/absoluteValuesSumMinization/absoluteValuesSumMinization.py
import math
def absoluteValuesSumMinimization(a):
    return a[math.floor(len(a) // 2)] if (len(a) % 2) else a[(len(a) // 2) - 1]


print(absoluteValuesSumMinimization([2, 4, 7]));
print(absoluteValuesSumMinimization([2, 4, 6, 7]));
print(absoluteValuesSumMinimization([2, 4, 6, 6, 7]));
print(absoluteValuesSumMinimization([2, 4, 6, 6, 7, 8]));